src/shared/mount-setup.h

```diff
@@ -10,3 +10,5 @@ int mount_cgroup_controllers(void);
 
 bool mount_point_is_api(const char *path);
 bool mount_point_ignore(const char *path);
+
+bool memory_recursiveprot_supported(void);
```

Comment:
why export this?

Reply:
If I dont export this, there will be building errrors which say:
../src/shared/mount-setup.c:90:11: error: ‘check_cg_feature_supported’ undeclared here (not in a function)
and
../src/shared/mount-setup.c:151:6: error: no previous prototype for ‘check_cg_feature_supported’ [-Werror=missing-prototypes]
see:https://github.com/systemd/systemd/actions/runs/5385453114/jobs/9774480321?pr=28154

Reply:
I think it should be declared before used. 
The function is after the mount table, so I think I need declare it in header file.

Reply:
just move the function before the mount table?

(or add a static prototype before the table? – but honetsly ust move the function up)

Reply:
fixed.